README.md

```diff
@@ -33,7 +28,7 @@ To use this library in your Spigot plugin, you should add it as a dependency. He
      <dependency>
         <groupId>com.github.btror</groupId>
         <artifactId>mc-pathfinding</artifactId>
-        <version>1.3.28</version>
+        <version>"insert package version"</version>
```

Comment:
it'd be nice to have a script to auto update this in the readme later. It'd probably need to be run with a workflow, this placeholder should be fine.